### PR TITLE
Fix issue #2: Add ESLint

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,11 +4,19 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "./node_modules/eslint/bin/eslint.js src/*/*.js"
```
Comment (mariolamacchia) [inline package.json]: Only `eslint` is enough: when running npm scripts all the node_modules bin are in the PATH

File: src/core/events.js [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-    handlers.forEach((h) => {
-        if (h.event === event) {
-            h.callback.apply(null, Array.prototype.slice.call(arguments, 1));
-        }
-    })
-}
+const fire = (event, ...args) => {
+  handlers.forEach((handler) => {
+    if (handler.event === event) {
+      handler.callback.apply(null, Array.prototype.slice.call(args, 1));
```
Comment (mariolamacchia) [inline src/core/events.js]: if you use `...args` you don't need to slice, it should be `apply(args)`

**Comment by mariolamacchia** [comment]
Did you fix all the eslint errors?

**Comment by ghost** [comment]
Yeah

File: src/core/events.js (hunk 2 above)
Comment (ghost) [reply src/core/events.js]: I'll fix it ASAP

File: package.json (hunk 1 above)
Comment (ghost) [reply package.json]: I'll fix it ASAP

**Comment by mariolamacchia** [comment]
There were still two files missing, but I fixed them. There are still a couple of errors in `scopa/index.js`, related to the rules `no-param-reassign` and `no-return-assign`. 

I haven't disabled the rules because they do make sense. When there is no simple way to fix the lint issue that always means there is a problem in the code. I'll leave you the fun :)

Free tip: carefully read the eslint docs and the airbnb guidelines ;)

**Pull request merged**